apis/python/src/tiledbsoma/io/registration/id_mappings.py

```diff
@@ -0,0 +1,81 @@
+from typing import Dict, List
+
+import anndata as ad
+import attrs
+import pandas as pd
+from typing_extensions import Self
+
+import tiledbsoma
+import tiledbsoma.logging
+
+
+@attrs.define(kw_only=True)
```

Comment:
Since this is a one-property class `kw_only` seems excessive to me here, but it’s fine to keep it this way if you prefer.

Reply:
@thetorpedodog it has one property _today_ but it may have two _tomorrow_ and your code-review guidance on #1554 was excellent. Let us not in our over-eager parsimony deprive future maintainers of _this_ file from excellent advice implement on _that_ file. Let us emplace good design patterns from the get-go. Not to mention simple consistency -- _all_ the `@attrs` classes in this module will be `kw_only`, not a mix of some-yes-some-no.